Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM openjdk:11
+#Copy project to docker container
+COPY ./ ./
+#Change directory to main directory
+WORKDIR src/main/java/in/cognitron/lms/
+#Compile java code 
+RUN javac -cp "../../../../../../lib/*" Server.java
```

Comment:
Updated Dockerfile

Reply:
I meant to stay at the root and use environment variables for both the source and destination directories.  Otherwise, your updated version is really just the same as it was before.